src/main/java/pl/stormit/eduquiz/authenticator/controllers/AuthenticationViewController.java

```diff
@@ -0,0 +1,61 @@
+package pl.stormit.eduquiz.authenticator.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import pl.stormit.eduquiz.quizcreator.domain.user.UserService;
+import pl.stormit.eduquiz.quizcreator.domain.user.dto.UserRequestDto;
+
+@Controller
+@RequestMapping("/register")
+@AllArgsConstructor
+public class AuthenticationViewController {
+
+
+    private final UserService userService;
+
+    @GetMapping()
+    public String registerUser(final Model model, Error error) {
+        model.addAttribute("userRequestDto", new UserRequestDto(null,
+                null, null, null, null, null, null));
```

Comment:
Can we do something about this constructor and the "null" parameters?

Reply:
Due to the fact, that userRequestDto is "record" type, probably not, at least from what we tried